osu.Game/Graphics/UserInterface/FocusedTextBox.cs

```diff
@@ -30,6 +30,8 @@ public bool HoldFocus
             }
         }
 
+        public override bool HandleKeyboardInput => HoldFocus || base.HandleKeyboardInput;

```

Comment:
A comment is warranted here, that we need `HandleKeyboardInput` to be true so we receive focus.